cktap/proto.py

```diff
@@ -204,6 +205,23 @@ def get_xpub(self, cvc, master=False):
         xpub = st['xpub']
         return encode_base58_checksum(xpub)
 
+    def get_xpub_at_path(self, cvc, bip32_path: str):
+        assert self.is_tapsigner
+        int_path = str2path(bip32_path)
+        check_bip32_path(int_path)  # raises ValueError if hardened after non-hardened
+        hardened, non_hardened = split_bip32_path(int_path)
+        if not non_hardened:
+            self.set_derivation(path=bip32_path, cvc=cvc)
+            return self.get_xpub(cvc)
+        else:
+            str_path = path2str(hardened)
```

Comment:
kill this confusing local var by fixing path2str (when called from set_derivation) so that it handles int paths